plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -793,6 +808,11 @@ private SliceWriteFunction oracleCharWriteFunction()
         return SliceWriteFunction.of(Types.NCHAR, (statement, index, value) -> statement.unwrap(OraclePreparedStatement.class).setFixedCHAR(index, value.toStringUtf8()));
     }
 
+    private String toTargetType(ConnectorSession session, Type type)
```

Comment:
Not sure if `toTargetType` fits to `...Client` class.
Why not to inline this method?